### DISCUSSION TITLE
Does the packet support ECG analysis during motion?

### DISCUSSION THREAD

**Discussion by Luo-XiuFeng** [neuropsychology/NeuroKit#542] [Q&A] [answered]
Does the packet support ECG analysis during motion?Such as ecg signals collected while running or cycling to calculate heart rate?

**Comment by DominiqueMakowski** [comment, marked as answer]
There is no categorical difference between an ECG during motion and at rest, motion is just likely to add noise to the signal. 

NeuroKit has various more-or-less robust methods for cleaning the signal, so it might work as long as your signal is not too noisy (don't forget to inspect the results visually by plotting).
